app/models/knight.rb

```diff
@@ -1,2 +1,19 @@
 class Knight < Piece
+  def obstructed_diagonally?
+    false
```

Comment:
This looks good to me, I think that when we call this on a Knight it should work. But I'm not sure if this would work if we called it on a `Piece` with `type: Knight`. Maybe? Would want to play around in rails console a bit to find out.